.github/IMPROVEMENT_PROPOSAL.md

```diff
@@ -0,0 +1,15 @@
+**ETF Improvement Proposal (EIP)**
+An EIP is a GitHub issue which proposes an improvement of the ETF validator.
+It must follow the template below and provide information for all 4 sections.
+
+# Background and Motivation:
+Explain the problem behind this proposal.
```

Comment:
We could also ask for implementation evidence or links to (external discussions) illustrating the issue.